tutorials/object_gcr_4_photoz.ipynb

```diff
@@ -0,0 +1,264 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are 72 objects in this patch, and there are 101 bins in the photo-z PDF, so this 2D array has a shape of (72, 101). Note how the 2D array is orientied."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data['photoz_pdf'].shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's plot 10 PDFs. To get an array of bin center values, you can access the `pz_pdf_bin_centers` attribute. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for pdf in data['photoz_pdf'][:10]:\n",
+    "    plt.plot(cat.photoz_pdf_bin_centers, pdf);\n",
+    "\n",
+    "plt.xlabel('$z$');\n",
+    "plt.ylabel('$p(z)$');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example\n",
+    "\n",
+    "Now that we have learned all the access methods, let's try to work out an example!\n",
+    "\n",
+    "First of all, let's define a set of reasonable cuts to give us galaxies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cuts = [\n",
+    "    GCRQuery('extendedness > 0'),     # Extended objects\n",
+    "    GCRQuery((np.isfinite, 'mag_i')), # Select objects that have i-band magnitudes\n",
+    "    GCRQuery('clean'), # The source has no flagged pixels (interpolated, saturated, edge, clipped...) \n",
+    "                       # and was not skipped by the deblender\n",
+    "    GCRQuery('snr_i_cModel > 10'),    # SNR > 10\n",
+    "    GCRQuery('mag_i_cModel < 22'),  # cModel imag brighter than 22\n",
+    "    GCRQuery('mag_i_cModel > 20'),  # cModel imag fainter than 20 (exclude super bright objects)\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's make some plots!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = cat.get_quantities(['photoz_mode', 'mag_g_cModel', 'mag_r_cModel', 'mag_i_cModel'], filters=cuts, native_filters=['tract==4850'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.hist(data['photoz_mode'], 50);"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.scatter(data['mag_g_cModel'] - data['mag_r_cModel'],\n",
+    "            data['mag_r_cModel'] - data['mag_i_cModel'],\n",
+    "            c=data['photoz_mode'], s=4, vmin=0, vmax=1);\n",
+    "\n",
+    "plt.xlim(-1, 3);\n",
+    "plt.ylim(-0.5, 2);\n",
+    "plt.xlabel('$g-r$');\n",
+    "plt.ylabel('$r-i$');\n",
+    "plt.colorbar(label='$z$');"
+   ]
```

Comment:
It might be worthwhile putting a small bit of text here to talk about what you are seeing (to make the tutorial more informative), like "we expect the gradient in the mode as a function of r-i and g-r..." etc.?)

Reply:
I only partially addressed this. I added a "food for thought" comment (which is a lazy way to not add the actual explanation). If we want to add a short explanation of the color-color plot, I should let @sschmidt23 do it!

Reply:
Great, I think this is fine - just making people think about it is a great addition to the notebook!